explorer_backend/tsconfig.json

```diff
@@ -30,6 +30,6 @@
   "ts-node": {
     "esm": true
   },
-  "exclude": ["node_modules"],
-  "include": ["./**/*.ts", "package.json"],
+  "exclude": ["node_modules", "dist"],
```

Comment:
outDir should be excluded by default by typescript

Reply:
I've copied it from niljs :)
But fine, I'll remove it.

Reply:
Wait, I can't remove it. Because I don't use outDir, I use outFile